js/sandbox.js

```diff
@@ -33,7 +33,7 @@ var Sandbox =  {
                 }         
             },
             listen : function (evts) {
-                if (core.is_obj(evts)) {
+                if (core.is_obj(evts) || core.is_arr(evts)) {
```

Comment:
Nope sandbox is mine and so is core though the base of it comes from https://code.tutsplus.com/tutorials/writing-modular-javascript--net-14746